config/routes.rb

```diff
@@ -1,22 +1,24 @@
 Rails.application.routes.draw do
   root "home#index"
-  post "oauth/callback" => "oauths#callback"
-  get "oauth/callback" => "oauths#callback"
-  get "oauth/:provider" => "oauths#oauth", as: :auth_at_provider
-
-  get "review" => "review#index"
-
-  get "login" => "sessions#new"
-  put "review_card" => "review#review_card"
-  post "logout" => "sessions#destroy", as: :logout
+  scope :home , module: 'home' do
```

Comment:
Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.